modules/mariadb/src/test/java/org/testcontainers/junit/mariadb/SimpleMariaDBTest.java

```diff
@@ -144,6 +144,18 @@ public void testEmptyPasswordWithRootUser() throws SQLException {
         }
     }
 
+    @Test
+    public void testFondationImages() throws SQLException {
+        try (MariaDBContainer<?> mysql = new MariaDBContainer<>("quay.io/repository/mariadb-foundation/mariadb-devel?tab=tags&tag=11.7").withUsername("root")) {
```

Comment:
any specific reason to declare `.withUsername("root")`? It's great to have a test, however, quay has had many outages in the past https://status.redhat.com/history. Not sure we want to cause those failures on our CI system.

Reply:
Filtering to the [quay.io registry component](https://status.redhat.com/history?filter=fvnvxz75h3m2) there seems to be [one incident in August 2024](https://status.redhat.com/incidents/hp840dt7cgvw) since February 2024 when they migrated there. The [March 2024](https://status.redhat.com/incidents/qh68rjfg6xs6) incident looked like deployment rather than pulls.

Did you experience more than these?

There's currently about 300-400 pulls per day on the mariadb-devel image so I'm just hoping its reliable enough for everyone.